code/src/Components/header.css

```diff
@@ -0,0 +1,39 @@
+header {
+  /* width: 90%; */
+  display: flex;
+  justify-content: center;
+  padding: 0 20%;
+}
+
+header h1 {
+  font-size: 3.2em;
+  color: #ffff;
+  margin-bottom: 5%;
+  text-shadow: 5px 5px black;
```

Comment:
The effect looks really great!! I initially thought you used a different font that was that way.